libs/ledger-live-common/src/families/aptos/types.ts

```diff
@@ -38,34 +38,21 @@ export type AptosAddress = {
   path: string;
 };
 
-export interface TransactionEstimate {
+export interface TransactionOptions {
   maxGasAmount: string;
   gasUnitPrice: string;
-  sequenceNumber?: string;
-  expirationTimestampSecs?: string;
 }
 
-export type TransactionOptions = {
-  maxGasAmount: string;
-  gasUnitPrice: string;
-  sequenceNumber?: string;
-  expirationTimestampSecs?: string;
-};
-
 export type TransactionErrors = {
   maxGasAmount?: string;
   gasUnitPrice?: string;
-  sequenceNumber?: string;
-  expirationTimestampSecs?: string;
 };
 
 export type Transaction = TransactionCommon & {
   mode: string;
   family: "aptos";
   fees?: BigNumber | null;
-  options: TransactionOptions;
-  estimate: TransactionEstimate;
-  firstEmulation: boolean;
+  estimate: TransactionOptions;
```

Comment:
here you call it `estimate` and the interface is `TransactionOptions`, perhaps change to `options` or `TransactionEstimate`